### ISSUE TITLE
Finish Implementing Presets Backend

### ISSUE THREAD

**Issue by IamDH4** [djfun/audio-visualizer-python#26]
Waiting on UI implementation, most of the backend code is in-place thanks to @tassaron2. Remove button does not currently have a method associated with it.

**Comment by tassaron2** [comment]
All preset functionality should be working as of 2ad14b7d6ca9216bcdc72c5e13937fcbccc887a3. There are methods now for creating, deleting, renaming (saving+deleting), opening, exporting, and importing presets. Export and import use the same file format as project files (called avl instead of avp) and are used to transfer presets to other systems.

Only feature remaining to add is exporting and importing multiple presets in one avl file. The file format already supports doing this, and presets exported with this version will be forward compatible. So adding it to the backend will be simple, but a GUI dialog for selecting/deselecting presets out of an avl file must be designed as well.

**Comment by IamDH4** [comment]
@tassaron2 
> Only feature remaining to add is exporting and importing multiple presets in one avl file.

Should we wait on implementing this until after we hit our [first milestone](https://github.com/djfun/audio-visualizer-python/milestone/1)?

**Comment by tassaron2** [comment]
Yes, I agree we can wait to implement multiple preset functionality until a future milestone. At that time we can create a new issue.
